libnavigation-base/gradle.properties

```diff
@@ -1,3 +1,4 @@
 POM_ARTIFACT_ID=base
+POM_VERSION_NAME="${rootDir}/gradle.properties/POM_CORE_VERSION_NAME"
```

Comment:
This probably fails, doesn't it? Let's use `POM_VERSION_NAME=1.0.0-SNAPSHOT` in every core module for now and figure out the scripting in a follow-up PR.